src/SUMMARY.md

```diff
@@ -150,7 +151,6 @@
 - [Borrowing](borrowing.md)
   - [Borrowing a Value](borrowing/shared.md)
   - [Borrow Checking](borrowing/borrowck.md)
-  - [Borrow Errors](borrowing/examples.md)
```

Comment:
I think this is also lost by mistake?

Reply:
Yes.My system sucks some time alot sorry.